src/ui/Pricing/SubscriptionSettings/SubscriptionCard/Card.svelte

```diff
@@ -39,10 +39,19 @@
     </div>
   {/if}
 
-  <div class="actions mrg-xl mrg--t">
+  {#if billing && !shouldHideBillingInfo}
+    <div class="billing txt-m c-waterloo txt-right">
+      {billing}:
+      <h4 class="h4 c-black">
+        {price}{#if discount}/mo{/if}
+      </h4>
+    </div>
+  {/if}
+
+  <div class="actions row v-center mrg-xl mrg--t">
     <svelte:element
       this={link ? 'a' : 'button'}
-      class="btn-1 v-center"
+      class="btn-1 v-center body-2"
```

Comment:
`body-2` class breaks existing desktop UI. Please, make sure it's consistent with Figma design